operator/v1/types_network.go

```diff
@@ -191,3 +218,19 @@ const (
 	// for sophisticated network isolation and segmenting. This is the default.
 	SDNModeNetworkPolicy SDNMode = "NetworkPolicy"
 )
+
+// MACVLANMode is the Mode of macvlan. The value should be decapitalize
+// due to the CNI plugin
+type MACVLANMode string
+
+const (
+	// MACVLANModeBridge is the macvlan with thin bridge function.
+	MACVLANModeBridge MACVLANMode = "bridge"
+	// MACVLANModePrivate
+	MACVLANModePrivate MACVLANMode = "private"
+	// MACVLANModeVEPA is used with Virtual Ethernet Port Aggregator
+	// (802.1qbg) swtich
```

Comment:
The "Q" in "802.1Qbg" is capitalized, but I'm not sure about the doc strings here anyway; they're kinda useless unless you already know what they mean, but of course, they're also useless if you do already know what they mean. I think it would be better to have a pointer to full documentation (kernel macvlan docs? "man ip-link"?) from the `MACVLANMode` doc and then not say anything here...

Reply:
Okey. I've added to mention "man ip-link".

Reply:
(The "q" here is still capitalized wrong, and "switch" is misspelled, but you could also just delete that line)